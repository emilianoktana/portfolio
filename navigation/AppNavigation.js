import { createBottomTabNavigator } from 'react-navigation'

import HomeScreen from '../screens/HomeScreen'
import InvestmentsScreen from '../screens/InvestmentsScreen'

export default createBottomTabNavigator(
    {
      Home: HomeScreen,
      Portfolio: InvestmentsScreen
    }
  )