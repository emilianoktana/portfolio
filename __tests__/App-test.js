import 'react-native';
import React from 'react';

import AppNavigation from '../navigation/AppNavigation'

import { Provider } from 'react-redux'
import store from '../store'

import renderer from 'react-test-renderer';

it('Renders correctly', () => {
  const renderer = renderer.create(
    <Provider store={store}>
        <AppNavigation />
    </Provider>
    ).toJSON();
  expect(renderer).toBeTruthy()
});