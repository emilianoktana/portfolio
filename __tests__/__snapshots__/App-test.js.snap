// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "overflow": "hidden",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      collapsable={false}
      pointerEvents="auto"
      removeClippedSubviews={false}
      style={
        Array [
          Object {
            "flex": 1,
            "overflow": "hidden",
          },
          Array [
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
            Object {
              "opacity": 1,
            },
          ],
        ]
      }
    >
      <View
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <RCTSafeAreaView
          style={
            Object {
              "backgroundColor": "#dedede",
              "flex": 1,
            }
          }
        >
          <View
            style={
              Object {
                "backgroundColor": "#dedede",
                "flex": 1,
                "paddingVertical": 20,
              }
            }
          />
        </RCTSafeAreaView>
      </View>
    </View>
  </View>
  <View
    collapsable={undefined}
    onLayout={[Function]}
    pointerEvents="box-none"
    style={
      Object {
        "backgroundColor": "#fff",
        "borderTopColor": "rgba(0, 0, 0, .3)",
        "borderTopWidth": 0.5,
        "flexDirection": "row",
        "height": 49,
        "paddingBottom": 0,
        "paddingLeft": 0,
        "paddingRight": 0,
        "paddingTop": 0,
      }
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel="Home"
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "alignItems": "center",
            "flex": 1,
          },
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "flexDirection": "column",
            "justifyContent": "flex-end",
          },
          undefined,
        ]
      }
      testID={undefined}
    >
      <View
        style={
          Array [
            Object {
              "height": 29,
            },
            false,
            Object {
              "flex": 1,
            },
          ]
        }
      >
        <View
          collapsable={undefined}
          style={
            Object {
              "alignItems": "center",
              "alignSelf": "center",
              "height": "100%",
              "justifyContent": "center",
              "minWidth": 25,
              "opacity": 1,
              "position": "absolute",
              "width": "100%",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
          />
        </View>
        <View
          collapsable={undefined}
          style={
            Object {
              "alignItems": "center",
              "alignSelf": "center",
              "height": "100%",
              "justifyContent": "center",
              "minWidth": 25,
              "opacity": 0,
              "position": "absolute",
              "width": "100%",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
          />
        </View>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        collapsable={undefined}
        ellipsizeMode="tail"
        numberOfLines={1}
        style={
          Object {
            "backgroundColor": "transparent",
            "color": "tomato",
            "fontSize": 11,
            "marginBottom": 1.5,
            "textAlign": "center",
          }
        }
      >
        Home
      </Text>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel="Portfolio"
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "alignItems": "center",
            "flex": 1,
          },
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "flexDirection": "column",
            "justifyContent": "flex-end",
          },
          undefined,
        ]
      }
      testID={undefined}
    >
      <View
        style={
          Array [
            Object {
              "height": 29,
            },
            false,
            Object {
              "flex": 1,
            },
          ]
        }
      >
        <View
          collapsable={undefined}
          style={
            Object {
              "alignItems": "center",
              "alignSelf": "center",
              "height": "100%",
              "justifyContent": "center",
              "minWidth": 25,
              "opacity": 0,
              "position": "absolute",
              "width": "100%",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
          />
        </View>
        <View
          collapsable={undefined}
          style={
            Object {
              "alignItems": "center",
              "alignSelf": "center",
              "height": "100%",
              "justifyContent": "center",
              "minWidth": 25,
              "opacity": 1,
              "position": "absolute",
              "width": "100%",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
          />
        </View>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        collapsable={undefined}
        ellipsizeMode="tail"
        numberOfLines={1}
        style={
          Object {
            "backgroundColor": "transparent",
            "color": "gray",
            "fontSize": 11,
            "marginBottom": 1.5,
            "textAlign": "center",
          }
        }
      >
        Portfolio
      </Text>
    </View>
  </View>
</View>
`;
